src/js/components/awardv2/contract/AdditionalInfo.jsx

```diff
@@ -27,13 +27,23 @@ export default class AdditionalInfo extends React.Component {
     }
     render() {
         const awardData = this.props.overview;
+        const periodOfPerformanceData = awardData.periodOfPerformance || awardData.dates;
+        const parentAwards = awardData.parentAwardDetails;
```

Comment:
Related to my comment in the model - calling this `parentAward` or `parentAwardDetails` would improve clarity.